app/components/onboarding_consent/onboarding_consent.html.erb

```diff
@@ -3,7 +3,8 @@
     object: contributor,
     attr: :data_processing_consent,
     style: :leftAligned,
-    help: data_processing_consent_help.html_safe
+    help: data_processing_consent_help,
```

Comment:
Have you tried something like this?

```ruby
help = [
  data_processing_consent_help,
  Setting.onboarding_data_processing_consent_additional_info,
].join("\n\n").html_safe
```

Then you could probably pass it directly to the field component, without having to adjust them like this:

```ruby
c('field',
  ...
  help: simple_format(help),
  ...
)
```

Reply:
Yeah, I started off by doing that, but then I thought that the client might want to use multiple paragraphs, but I had forgotten about `simple_format`, so I think it should work. I'll give it a go. Thanks for the idea